packages/gatsby-image/src/index.js

```diff
@@ -260,9 +260,13 @@ class Image extends React.Component {
       itemProp,
     } = convertProps(this.props)
 
-    const shouldReveal = this.state.imgLoaded || this.state.fadeIn === false
-    const shouldFadeIn = this.state.fadeIn === true && !this.state.imgCached
-    const durationFadeIn = `0.5s`
+    const shouldReveal =
+      this.state.imgLoaded || Boolean(this.state.fadeIn) === false
+    const shouldFadeIn =
+      Boolean(this.state.fadeIn) === true && !this.state.imgCached
```

Comment:
Changing the type of an option and relying on coercion feels like a bad idea? Should we add a separate _opt-in_ `prop` called `durationFadeIn` (that defaults to 0.5s)?

Reply:
ooo yeah, I like this better 👍 

They are separate ideas so splitting makes sense + better backwards compatibility

Reply:
Sure, I'll do it !